packages/launch/src/hooks/use-site.ts

```diff
@@ -15,11 +15,16 @@ export function useSite() {
 	const site = useSelect( ( select ) => select( SITE_STORE ).getSite( siteId ) );
 	const launchStatus = useSelect( ( select ) => select( SITE_STORE ).isLaunched( siteId ) );
 
+	const isLoading: boolean = useSelect( ( select ) =>
+		select( 'core/data' ).isResolving( SITE_STORE, 'getSite', [ siteId ] )
```

Comment:
Is there a reason for not using `isFetchingSite` selector from `SITE_STORE`? This applies also to https://github.com/Automattic/wp-calypso/pull/47528/files#diff-8de21fd6530118850c1d099df9447b90bf23dd1fce2871d58da41634b06f0c7dR21

Reply:
Didn't know about this one!

Reply:
I tried it and it's never `true` 🤷

Reply:
You're right. Added the correct reducer in https://github.com/Automattic/wp-calypso/pull/47554/files#diff-df112a4c704268086d2d3100bede88722f9193c88c107359dc4c401ccf97834fR60